core/app/models/concerns/spree/has_image_alt_text.rb

```
@@ -0,0 +1,20 @@
+module Spree
+  module HasImageAltText
+    extend ActiveSupport::Concern
+
+    included do
+      # Define the image_alt method when the concern is included
+      define_method :image_alt do
+        return preferred_image_alt if preferred_image_alt.present?
+        
```

RuboCop (1.76.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)